pages/ui-components/packages.md

```diff
@@ -131,24 +131,23 @@ Using only package source would result in the following:
 @forward "uswds-helpers";
 ```
 
-If you suffer from slow compiles, it may be worth experiementing with source forwarding. 
+If you suffer from slow compiles, it may be worth experimenting with source forwarding. 
 
 ## Package Sass requirements: Load paths
 Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include the following load paths:
 
 ```js
-"./node_modules/@uswds",
+"./node_modules/@uswds/uswds",
```

Comment:
Flagging because it is that time of day and I need a sanity check on these include paths

Reply:
I think I may remove this first reference. It _is_ intended to reference the `@uswds` directory in order to support — down the road — component packages published into that org scope... like `@uswds/usa-banner`. But this functionality is not here yet, so it's probably more confusing than it needs to be

Reply:
```suggestion
```

Reply:
Removed in b05e3d4baddb9d5940f7ee911009897b7410b6b0